confidant/settings.py

```diff
@@ -604,6 +605,15 @@ def str_env(var_name, default=''):
 # last saw a credential pair
 ENABLE_SAVE_LAST_DECRYPTION_TIME = bool_env('ENABLE_SAVE_LAST_DECRYPTION_TIME')
 
+# Add any certificate authorities
+if str_env('SERVICE_INSTANCE') == 'development':
```

Comment:
`SERVICE_INSTANCE` is lyft specific.  I don't think we need this if clause?

Reply:
Good point, removing